### PR TITLE
Added Table Component consistency for Patient Address

### PR DESCRIPTION
**Pull request by GangulaMounika**
## Description
Table Component consistency for Patient Address

## Tickets

* https://cdc-nbs.atlassian.net/browse/CNFT1-1711

## Steps to verify
1. Provide steps here
2. Log in
3. Navigate to advanced search
4. Search for any patient
5. Navigate to patient profile and navigate to Demographics tabs and observe the Address table functioning as expected.

## Checklist before requesting a review
- [ ] PR focuses on a single story
- [ ] Code has been fully tested to meet acceptance criteria
- [ ] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [ ] All new functions/classes/components reasonably small
- [ ] Functions/classes/components focused on one responsibility
- [ ] Code easy to understand and modify (clarity over concise/clever)
- [ ] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [ ] PR does not contain hardcoded values (Uses constants)
- [ ] All code is covered by unit or feature tests

### CODE REVIEW

**Review by achapm — DISMISSED** [review]
👍

File: apps/modernization-ui/src/pages/patient/profile/addresses/AddressesTable.tsx [hunk 1]
```diff
@@ -278,59 +277,78 @@ export const AddressesTable = ({ patient }: Props) => {
...
+     * @return {TableBody}
+     */
+    const generateTableRow = (address: PatientAddress, index: number): TableBody => {
+        return {
+            id: index,
+            tableDetails: [
+                {
+                    id: 1,
+                    title: (
+                        <span className={`font-sans-1xs table-data ${tableHead[0].sort !== 'all' && 'sort-td'}`}>
```
[the remark this answers is not in the document]
Comment (GangulaMounika) [reply apps/modernization-ui/src/pages/patient/profile/addresses/AddressesTable.tsx:294]: Nice Catch, will take a look and fix it Adam!

File: apps/modernization-ui/src/pages/patient/profile/addresses/AddressesTable.tsx [hunk 2]
```diff
@@ -278,59 +276,73 @@ export const AddressesTable = ({ patient }: Props) => {
...
+     * @param {number} index, index of the array item
+     * @return {TableBody}
+     */
+    const generateTableRow = (address: PatientAddress, index: number): TableBody => {
+        return {
+            id: index,
+            tableDetails: [
+                {
+                    id: 1,
+                    title: address?.asOf ? (
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/pages/patient/profile/addresses/AddressesTable.tsx:292]: question: Are the extra `<span>` elements required?  The `font-sans-1xs` class is setting the font-size should be standard for all tables and the `table-data` class is not used by the `TableComponent`.

Comment (GangulaMounika) [reply apps/modernization-ui/src/pages/patient/profile/addresses/AddressesTable.tsx]: Oh I haven't changed the template much. Fix all those now. Thanks

**Review by adamloup-enquizit — APPROVED** [review]

**Pull request merged**